src/System.Private.CoreLib/src/System/Internal.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
this file was very weird because it calls methods with null which don't expect null because it's meant only for some AOT optimizations. I might have made some errors here - I wasn't sure how to do AOT build locally

Reply:
Can you just instead mark the file as `#nullable disable`?

Reply:
yes, good idea